openapi/paths/tracking@webhooks@{id}@resend.yaml

```
@@ -5,9 +5,10 @@ post:
     - Users
   tags:
     - Tracking
-  summary: Schedules resend webhook attempt
+  summary: Schedule a resend webhook attempt
```

Dot is missing at the end. However, should not it be `schedules` instead of `schedule`?

```suggestion
  summary: Schedule a resend webhook attempt.
```

---

We shouldn't start [summaries](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#summaries) with a plural or end them with a period (I think you may have misread this one as an [operation description](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#operations))